README.md

```diff
@@ -26,7 +26,7 @@ The directories also have no explicit field for parent directories, so one can l
 This allows for graph structures inside a file system, which a DFA basically is.
 
 ### Q: Should I use this <del>in production</del> anywhere?
-A: No, but I can't stop you.
+A: Yes
```

Comment:
Pull request looks mostly fine, but I'm not quite happy with this line change.
The version of this program is only 0.1.0, which by SemVer can not be considered stable.
It's also quite a big change to the documentation.

Reply:
Fixed in new commit